config/reasoner/dl2op/main-info/rrn.n3

```diff
@@ -0,0 +1,62 @@
+PREFIX : <#>
+PREFIX adms: <http://www.w3.org/ns/adms#>
+PREFIX adres: <https://data.vlaanderen.be/ns/adres#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX crypto: <http://www.w3.org/2000/10/swap/crypto#>
+PREFIX e: <http://eulersharp.sourceforge.net/2003/03swap/log-rules#>
+PREFIX ex: <http://example.org/test#>
+PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+PREFIX func: <http://www.w3.org/2007/rif-builtin-function#>
+PREFIX generiek: <https://data.vlaanderen.be/ns/generiek#>
+PREFIX log: <http://www.w3.org/2000/10/swap/log#>
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX org: <http://www.w3.org/ns/org#>
+PREFIX persoon: <https://data.vlaanderen.be/ns/persoon#>
+PREFIX persoon_loket: <http://data.vlaanderen.be/ns/persoon#>
+PREFIX skos:<http://www.w3.org/2004/02/skos/core#>
+PREFIX string: <http://www.w3.org/2000/10/swap/string#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+
+
+# Rijksregisternummer
+# Extend identifiers to match OP's model
+# <http://example.org/tst#subject> skos:notation "00010100105".
+{
+    ?identifier skos:notation ?notation.
+    ?notation string:notEqualIgnoringCase "Rijksregisternummer".
+    ?identifier log:uri ?identifierString.
+    (?identifierString "STRUCTURED_IDENTIFIER")  string:concatenation ?hashInput.
+    ?hashInput crypto:md5 ?uuid.
+    ("http://data.lblod.info/id/gestructureerdeIdentificatoren/" ?uuid) string:concatenation ?structuredIdString.
+    ?structuredIdentifier log:uri ?structuredIdString.
+
+    {
+      ?identifier skos:notation ?notation.
+    } e:becomes {
+      ?identifier
+          generiek:gestructureerdeIdentificator ?structuredIdentifier ;
+          skos:notation "Rijksregisternummer" .
+    }.
+
+} => {
+  ?identifier
```

Comment:
Here I think that the type is missing, at least I don't see it in the db when I consume it !
( `http://www.w3.org/ns/adms#Identifier` )

Reply:
Correct. Well spotted :) I've now added it to the private-info query.